docs/pages/next-steps/additional-resources.md

```diff
@@ -2,45 +2,39 @@
 title: Additional resources
 ---
 
-The following resources are useful for learning Expo and some of the projects that it depends on.
+The following resources are useful for learning about Expo tooling and services.
 
 ## Expo Blog
 
 - [Exposition](https://blog.expo.dev/) - our official blog, where we post release notes every month and other Expo related content at random intervals.
 
-## Courses using Expo
+## GitHub
 
-- [Stanford CS 47SI Cross-Platform Mobile Development](https://web.stanford.edu/class/cs47si/) (free)
-- [University of Helsinki - Full Stack Open](https://fullstackopen.com/en/part10) (free)
-- [React Europe - Introduction to React Native Workshop videos on YouTube](https://www.youtube.com/playlist?list=PLCC436JpVnK2RFms3NG9ubPToWCNbMLbT) (free)
-- [Udacity - React Nanodegree](https://www.udacity.com/course/react-nanodegree--nd019) (paid)
+- [expo/expo](https://github.com/expo/expo): Expo Go, SDK, Docs, and the Expo CLI.
+- [expo/eas-cli](https://github.com/expo/eas-cli): Fastest way to build, submit, and update iOS and Android apps.
+- [expo/examples](https://github.com/expo/examples): Integrations and other examples.
+- [expo/config-plugins](https://github.com/expo/config-plugins): Expo Config Plugins for working with third-party packages.
+- [expo/snack](https://github.com/expo/snack): Build apps from the browser.
+- [expo/vscode-expo](https://github.com/expo/vscode-expo): VS Code extension for working with Expo tools.
+- [expo/fyi](https://github.com/expo/fyi): Troubleshooting guides for Expo tools and services.
 
+## Documentation
+
+- [React Native](https://reactnative.dev/docs/getting-started)
+- [React](https://reactjs.org/docs/getting-started.html)
+- [Metro bundler](https://facebook.github.io/metro/)
+- [Hermes engine](https://hermesengine.dev/)
+- [Apple's Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/guidelines/overview/)
 
 ## React Native
 
 - [React Native Express](http://www.reactnativeexpress.com/) - The best way to get started with React Native! It's a walkthrough the building blocks of React and React Native.
-- [Fullstack React Native](https://www.fullstackreact.com/react-native/) (book) - From the author of React Native Express: build 7 complete apps, covering React Native fundamentals and advanced topics in-depth.
-- [Official React Native Documentation](https://reactnative.dev/docs/getting-started)
-- [React Native Fundamentals](https://egghead.io/courses/react-native-fundamentals) (video course on Egghead.io)
+- [Intermediate React Native](https://frontendmasters.com/courses/intermediate-react-native/) - Paid course by Frontend Masters.
 - [Animating React Native UI Elements](https://egghead.io/courses/animate-react-native-ui-elements) (video course on Egghead.io)
```

Comment:
I think this outdated. I had a similar open issue in Linear but never returned to it. So have some ideas.

I was wondering if we could add Reanimated and RNGH docs and create a new section called "Gestures and Animations". Also, create a new section for Navigation and a link to Navigation.

```
## Animation and Gestures
- [React Native Reanimated](https://docs.swmansion.com/react-native-reanimated/docs)
- [React Native Gesture Handler](https://docs.swmansion.com/react-native-gesture-handler/docs/)

## Navigation
- [React Navigation](https://reactnavigation.org/docs/getting-started/)
```

What do you think?

Reply:
Seems fine, we also have standalone API guides for these too.

Reply:
Do you think we should link to those standalone API guides instead?